codeserver/ubi9-python-3.12/run-code-server.sh

```
@@ -0,0 +1,129 @@
+
+# Start nginx and supervisord
+run-nginx.sh &
+/usr/bin/supervisord -c /etc/supervisor/conf.d/supervisord.conf &
+
+# Add .bashrc for custom prompt if not present
+if [ ! -f "/opt/app-root/src/.bashrc" ]; then
+  echo 'PS1="\[\033[34;1m\][\$(pwd)]\[\033[0m\]\n\[\033[1;0m\]$ \[\033[0m\]"' > /opt/app-root/src/.bashrc
+fi
+
+# Initialize access logs for culling
+echo '[{"id":"code-server","name":"code-server","last_activity":"'$(date -Iseconds)'","execution_state":"running","connections":1}]' > /var/log/nginx/codeserver.access.log
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)